src/backends/tensorrt/protoUtils.cc

```diff
@@ -209,4 +212,51 @@ namespace dd
     return success;
   }
 
+  bool TRTReadProtoFromBinaryFile(const char *filename,
+                                  google::protobuf::Message *proto)
+  {
+    std::ifstream input(filename, std::ios::in | std::ios::binary);
+    if (!input.good())
+      return false;
+    bool success = proto->ParseFromIstream(&input);
+    return success;
+  }
+
+  namespace onnx_proto
+  {
+    // https://github.com/onnx/onnx/blob/master/onnx/onnx.proto
+    int findTopK(const std::string &source)
+    {
+      onnx::ModelProto net;
+      if (!TRTReadProtoFromBinaryFile(source.c_str(), &net))
+        {
+          return -1;
+        }
+
+      int node_count = net.graph().output_size();
+
+      for (int i = 0; i < node_count; ++i)
+        {
+          const auto &out = net.graph().output(i);
+          std::string name = out.has_name() ? out.name() : "";
+
+          if (name == "detection_out")
```

Comment:
Do we want this as parameter to the function or it does not make sense because it is so specialized ? 
If the function is highly specialized that for instance it's only for yolox or ssd models, do we want to modify it's name, like `findTopKYolox` ?

Reply:
In torch when we export the model we can set the name of the output to whatever we want. I chose "detection_out" because it was the output name used in deepdetect. So this name should be generic to all detection models, and not specific to Yolox

Reply:
ok, but that's external to DD code itself, so my opinion is that it is still best to have it as a parameter, it's a detail though.

Reply:
I made it a parameter